### PR TITLE
Release v0.5.4

### PR DESCRIPTION
**Pull request by jtherrmann**
TODO:

- [x] Confirm that JSON -> YAML conversion didn't break our Landsat SNS subscription
- [x] Include the latest S2 fix (WIP) as part of this release
- [x] Run the integration tests?

### CODE REVIEW

File: requirements-its_live_monitoring.txt [hunk 1]
```diff
@@ -1,6 +1,5 @@
-geopandas==0.14.4
+geopandas==1.0.0
 hyp3-sdk==6.1.0
 pandas==2.2.2
-pystac-client==0.7.7
-requests==2.31.0
```
Comment (jtherrmann) [inline requirements-its_live_monitoring.txt:5]: Weird, I don't remember removing the `requests` version pin, I wonder if that was accidental.

Comment (jtherrmann) [reply requirements-its_live_monitoring.txt:5]: I think I accidentally deleted it while resolving merge conflicts: https://github.com/ASFHyP3/its-live-monitoring/commit/73298cdc81301f335b1197ea35b80d09be8a2efb

File: requirements-its_live_monitoring.txt [hunk 2]
```diff
@@ -1,6 +1,5 @@
...
+pystac-client==0.8.2
```
Comment (jtherrmann) [inline requirements-its_live_monitoring.txt:4]:
```suggestion
requests==2.31.0
pystac-client==0.8.2
```

File: requirements-its_live_monitoring.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
-geopandas==0.14.4
+geopandas==1.0.0
 hyp3-sdk==6.1.0
 pandas==2.2.2
-pystac-client==0.7.7
 requests==2.31.0
```
Comment (jtherrmann) [inline requirements-its_live_monitoring.txt:4]:
```suggestion
requests==2.32.3
```

**Review by asjohnston-asf — APPROVED** [review]

**Pull request merged**